<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph</class>
 <widget class="QWidget" name="graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: beige;</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>831</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E28A52;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pagename">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>15</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: #0B1A36;</string>
   </property>
   <property name="text">
    <string>数据统计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #0B1A36;
border-radius: 15px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>返回主页</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/arrow_left.png</normaloff>:/icons/icons/arrow_left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>771</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="bigLayout">
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="graphLayout"/>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QComboBox" name="comboRange">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>今天</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>当前月份</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>今年</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
